_data/changelogs/tokens-spacing.yml

```
@@ -2,6 +2,19 @@ title: Spacing tokens
 type: token
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Updated token value of `0` to match other tokens by using `spacing-multiple()`
+    summaryAdditional: This corrects issues found when `units(0)` is used in a `calc()` function
```

Similar comment as above. Trying to keep it focused on what affects the user most. 
```suggestion
    summary: Fixed a bug that prevented `units(0)` from being used in a `calc()` function.
    summaryAdditional: This change updated the unit-less `0` token value to have a unit when needed.
```

---

Same, was this resolved?

---

Implemented @amyleadem's [recommendation ](https://github.com/uswds/uswds-site/pull/2127#discussion_r1288885434) in [75c154c](https://github.com/uswds/uswds-site/pull/2127/commits/75c154cb84b4135520e5a85d5ccea1fcddb35bb4)